validate-read-structure.html

```diff
@@ -0,0 +1,67 @@
+---
+---
+<!doctype html>
+<html lang="en">
+
+	<head>
+		{% include head.html title="Validate Read Structures" %}
+		<script type="text/javascript" src="https://code.jquery.com/jquery-1.12.4.js"></script>
+		<script type="text/javascript" src="js/validate-read-structure.js"></script>
+	</head>
+
+	<noscript>This page requires JavaScript. Please enable it in your browser settings.</noscript>
+
+	<body>
+		<div class="wrapper">
+			{% include header.html %}
+			<h2>Validate Read Structures</h2>
+			<section>
+				<p>This utility validates one or more <a href="https://github.com/fulcrumgenomics/fgbio/wiki/Read-Structures">Read Structures</a> for use with <a href="http://fulcrumgenomics.github.io/fgbio/">fgbio</a>.</p>
+
+				<p>Enter in a read structure below. Multiple read structures can be entered separated by a space.</p>
```

Comment:
I could not get multiple read structures separated by spaces to work.
Also probably "by a space" -> "by spaces".

Reply:
Fixed.